ci/aws-analytical-env/jobs/mirror-git-repo-s3.yml

```diff
@@ -0,0 +1,19 @@
+jobs:
+  - name: mirror-git-repo-s3
+    plan:
+      - get: chs-uc-lab-repo
+      - get: aws-common-infrastructure
+        trigger: false
+      - .: (( inject meta.plan.terraform-output-common ))
+        config:
+          params:
+            AWS_ACC: ((aws_account.development))
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
```

Comment:
what are these for, they don't appear to be used anywhere?

Reply:
removed them